src/views/CoreUnitAbout/components/Markdown/renderUtils.tsx

```diff
@@ -1,16 +1,21 @@
 import { styled } from '@mui/material';
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
 import './Markdown.module.scss';
 
+// Use uuid to generate unique keys
+const generateKey = (prefix: string) => `${prefix}-${uuidv4()}`;
```

Comment:
While UUIDs are unique, it is good if they come from the API as the ID will not change. This way it changes every time the app re-renders.

Reply:
you are right, done